src/sentry/static/sentry/app/components/quickTrace/utils.tsx

```diff
@@ -74,6 +74,7 @@ export function generateTraceTarget(
   const dateSelection = getParams(getTraceTimeRangeFromEvent(event));
 
   if (organization.features.includes('trace-view-summary')) {
+    // TODO(txiao): Should this persist the current query when going to trace view?
```

Comment:
Currently, when linking to the trace view, the query parameters are dropped because the query on a transaction does not necessarily apply to the trace.